### PR TITLE
Refactored risk statistics

### PR DESCRIPTION
**Pull request by micheles**
Another step towards https://bugs.launchpad.net/oq-engine/+bug/1416959. The tests are green: https://ci.openquake.org/job/zdevel_oq-risklib/478

### CODE REVIEW

File: openquake/risklib/scientific.py [hunk 1]
```diff
@@ -33,6 +34,27 @@
 from openquake.risklib import utils
 
 
+class Output(object):
+    """
+    A generic container of attributes. Only assets, loss_type, hid and weight
+    are always defined.
+    """
```
Comment (ptormene) [inline openquake/risklib/scientific.py]: please also describe the params in the docstring

File: openquake/risklib/scientific.py [hunk 2]
```diff
@@ -1066,8 +1112,32 @@ def normalize(self, loss_curves):
...
+        Build all statistics from a set of risk outputs.
+
+        :param all_outputs:
+            a non empty sequence of risk outputs referring to the same assets
+            and loss_type. Each output must have attributes assets, loss_type,
+            hid, weight, loss_curves and insured_curves (the latter is
+            possibly None).
+        :returns:
+            an Output object with the following attributes
+            (numpy arrays; the shape is in parenthesis):
```
Comment (ptormene) [inline openquake/risklib/scientific.py]: Please specify the meaning of N, R, P, Q, etc.

**Comment by ptormene** [comment]
Besides the docstrings, LGTM

File: openquake/risklib/scientific.py (hunk 1 above)
Comment (micheles) [inline openquake/risklib/scientific.py]: Done

File: openquake/risklib/scientific.py (hunk 2 above)
Comment (micheles) [inline openquake/risklib/scientific.py]: Done

**Pull request merged**